proxy/proxy.go

```diff
@@ -19,6 +21,8 @@ const (
 	prefixKey contextKey = "prefix"
 
 	PrefixHeader string = "X-Forwarded-Prefix"
+
+	TempoPrefixHeader string = "X-Scope-OrgID"
```

Comment:
Should this value come from `cfg.traces.tenantHeader` instead? In tempo-operator, we're starting the gateway with `--traces.tenant-header=x-scope-orgid`

Reply:
My intention was to forward the tenant to tempo in the header that tempo expects. which is defined here: https://grafana.com/docs/tempo/latest/operations/multitenancy/ 


> empo is a multi-tenant distributed tracing backend. It supports multi-tenancy through the use of a header: X-Scope-OrgID.

Reply:
I wonder why `cfg.traces.tenantHeader` is configurable in the first place then, if it's always a constant set to `X-Scope-OrgID`?